lang/python/python3-version.mk

```diff
@@ -8,12 +8,12 @@
 # Note: keep in sync with setuptools & pip
 PYTHON3_VERSION_MAJOR:=3
 PYTHON3_VERSION_MINOR:=6
-PYTHON3_VERSION_MICRO:=9
+PYTHON3_VERSION_MICRO:=10
 
 PYTHON3_VERSION:=$(PYTHON3_VERSION_MAJOR).$(PYTHON3_VERSION_MINOR)
 
 PYTHON3_SETUPTOOLS_PKG_RELEASE:=1
-PYTHON3_PIP_PKG_RELEASE:=2
+PYTHON3_PIP_PKG_RELEASE:=1
```

Comment:
Not sure this should change if `PYTHON3_PIP_VERSION` hasn't changed.

Reply:
You are right. I'm going to remove this change.

Reply:
Removed.